pkg/codec/encodings/binary/gen/main.go

```diff
@@ -33,22 +39,9 @@ func main() {
 			panic(err)
 		}
 
-		_, location, _, ok := runtime.Caller(0)
-		if !ok {
-			panic("cannot find location of file")
-		}
-		// We need to use strings.LastIndex instead of strings.Index because the
-		// file path in CI can contain the string "chainlink-common" multiple
```

Comment:
I still have an old 	`chainlink-relay/` dir, so this was failing for me. The new approach should be equivalent, without caring about directory names.